### ISSUE TITLE
Imagemap for dark_locker.jpg (knees) is not functioning as expected

### ISSUE THREAD

**Issue by Nikith-Kalu** [sanik-dev/Teichopsia#19]
The imagemap for knees uses an alpha mask but besides that it is very similar to other imagemaps. The issue is in regards to the dialogue box, as it is unable to continue past the first line. 

Testing and troubleshooting shows that the following quirks are present:

1. The dialogue is able to continue past the first line whenever the mouse is not hovering over the imagemap for knees
2. Removing the alpha mask and retaining standard coordinates does not resolve the issue
3. Removing the vent imagemap does not resolve the issue however its worth noting that the vent imagemap suffers from the same issue after coding the imagemap for the knees
4. Commenting out all logic and irrelevant code does not resolve issues, the only things that were left in the code was the imagemap and the label for knees yet issues still persisted.

In summary...

The issue is thought to be with the imagemap exclusive to the knees, regardless of whether alpha mask is used or not. Basic coordinates still cause errors with being unable to do anything so long as the mouse is hovering over the imagemap.

**Comment by Nikith-Kalu** [comment]
Reverted branch back to https://github.com/sanik-dev/Teichopsia/commit/8a720d330ca0b7c6bb837c47dfef90a57a13bf30

- Added alpha mask image into images dir

**Comment by Nikith-Kalu** [comment]
Circumvented issue by avoiding use of alpha masks and instead using rectangular specified regions. Issue will be reopened when development reaches a phase where alpha masks are necessary.
